requirements.txt

```
@@ -4,4 +4,5 @@ requests==2.22.0
 peewee==3.11.2
 PyJWT==1.7.1
 python-dotenv==0.10.3
-pylint==2.4.2
\ No newline at end of file
+pylint==2.4.2
+simple_chalk
```

Add this dependency in `setup.py` also and make the changes as suggested below

```suggestion
simple-chalk==0.1.0
```

---

Done